README.md

```diff
@@ -1,42 +1,67 @@
+
 # Jellyfish Android Client
+
 Android client library for [Jellyfish](https://github.com/jellyfish-dev/jellyfish).
 
 ## Documentation
+
 Documentation is available [here](https://jellyfish-dev.github.io/android-client-sdk/).
 
 ## Installation
+
 Add jitpack repo to your build.gradle:
+
 ```gradle
-	allprojects {
-		repositories {
-			...
-			maven { url 'https://jitpack.io' }
-		}
-	}
+ allprojects {
+  repositories {
+   ...
+   maven { url 'https://jitpack.io' }
+  }
+ }
 ```
 
 Add the dependency:
+
 ```gradle
-	dependencies {
-	  implementation 'com.github.jellyfish-dev:android-client-sdk:<<version>>'
-	}
+ dependencies {
+   implementation 'com.github.jellyfish-dev:android-client-sdk:0.2.0'
```

Comment:
I think we should stick to version-less installation guides
```suggestion
   implementation 'com.github.jellyfish-dev:android-client-sdk:<<version>>'
```

Reply:
we have release-it script, you can configure it to update automatically :3